addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/XFrameOptionScanRule.java

```diff
@@ -188,4 +189,23 @@ private String getMetaXFOEvidence(Source source) {
         }
         return null;
     }
+
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<Alert>();
+        for (VulnType vulnType : VulnType.values()) {
+            alerts.add(
+                    newAlert()
+                            .setName(getAlertElement(vulnType, "name"))
+                            .setRisk(Alert.RISK_MEDIUM)
+                            .setConfidence(Alert.CONFIDENCE_MEDIUM)
```

Comment:
Do you care that this doesn't account for:
https://github.com/zaproxy/zap-extensions/blob/7261748f7dfc5c4762d272eab880a439707b9778/addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/XFrameOptionScanRule.java#L116-L121

Is "Other Info" included in the docs?

Reply:
Oops I guess this should only be associated with line 199.

Reply:
No - they are just example alerts. I dont think we can document every possible variation.